patches/type_asserts.patch

```diff
@@ -2,6 +2,75 @@ diff --git a/internal/server/openapi/type_asserts.go b/internal/server/openapi/t
 index b001018..9907fbc 100644
 --- a/internal/server/openapi/type_asserts.go
 +++ b/internal/server/openapi/type_asserts.go
+@@ -28,14 +28,14 @@ func AssertArtifactCreateConstraints(obj model.ArtifactCreate) error {
+ 
+ // AssertArtifactCreateRequired checks if the required fields are not zero-ed
+ func AssertArtifactCreateRequired(obj model.ArtifactCreate) error {
+-	elements := map[string]interface{}{
+-		"artifactType": obj.ArtifactType,
+-	}
+-	for name, el := range elements {
+-		if isZero := IsZeroValue(el); isZero {
+-			return &RequiredError{Field: name}
+-		}
+-	}
++	// elements := map[string]interface{}{
++	// 	"artifactType": obj.ArtifactType,
++	// }
++	// for name, el := range elements {
++	// 	if isZero := IsZeroValue(el); isZero {
++	// 		return &RequiredError{Field: name}
++	// 	}
++	// }
```

Comment:
unfortunately the generation isn't perfect for these kinds of models. Patched like `MetadataValue`

Reply:
I would just "file this" under another limitation of openapi-codegen